### PR TITLE
Added textiles fabricator.

### PR DESCRIPTION
**Pull request by biscuitWizard**
Added suits to the fabricator.

### CODE REVIEW

File: mods/ascent/_ascent.dme [hunk 1]
```diff
@@ -31,4 +31,11 @@
 #include "structures\ship_furniture.dm"
 #include "turfs\ship.dm"
 // END_INCLUDE
-#endif
\ No newline at end of file
+#endif
+// BEGIN_INTERNALS
```
Comment (comma) [inline mods/ascent/_ascent.dme]: some garbig here

File: code/modules/clothing/spacesuits/breaches.dm [hunk 2]
```diff
@@ -25,14 +25,20 @@
...
 	var/can_breach = 1                      // Set to 0 to disregard all breaching.
 	var/list/breaches = list()              // Breach datum container.
 	var/resilience = 0.2                    // Multiplier that turns damage into breach class. 1 is 100% of damage to breach, 0.1 is 10%. 0.2 -> 50 brute/burn damage to cause 10 breach damage
 	var/breach_threshold = 3                // Min damage before a breach is possible. Damage is subtracted by this amount, it determines the "hardness" of the suit.
 	var/damage = 0                          // Current total damage. Does not count patched breaches.
 	var/brute_damage = 0                    // Specifically brute damage. Includes patched punctures.
 	var/burn_damage = 0                     // Specifically burn damage. Includes patched burns.
+	origin_tech = "{'materials':3, 'engineering':3}"
+	material = MAT_PLASTIC
+	matter = list(
```
Comment (comma) [inline code/modules/clothing/spacesuits/breaches.dm]: that's a pretty odd place to add matter vars at, why not at `code\modules\clothing\spacesuits\spacesuits.dm`?

File: mods/ascent/_ascent.dme (hunk 1 above)
Comment (biscuitWizard) [reply mods/ascent/_ascent.dme]: It's being auto-added by DM plugin oddly.

File: code/modules/clothing/spacesuits/breaches.dm (hunk 2 above)
Comment (biscuitWizard) [reply code/modules/clothing/spacesuits/breaches.dm:37]: We spoke about making clothing printable, so added it to base /suit

Comment (biscuitWizard) [reply code/modules/clothing/spacesuits/breaches.dm]: At /obj/item/clothing/suit/space? What's odd about it?

Comment (comma) [reply code/modules/clothing/spacesuits/breaches.dm]: because it's not its real definition, but a partial override for breach specific vars?

Comment (biscuitWizard) [reply code/modules/clothing/spacesuits/breaches.dm]: That's crazy! Disadv to using search to find definitions sometimes.

File: mods/ascent/_ascent.dme (hunk 1 above)
Comment (MistakeNot4892) [reply mods/ascent/_ascent.dme]: You need to manually remove it after opening the DME in DM

File: code/modules/clothing/head/helmet.dm [hunk 3]
```diff
@@ -24,6 +24,9 @@
 	max_heat_protection_temperature = HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
 	siemens_coefficient = 0.7
 	w_class = ITEM_SIZE_NORMAL
+	matter = MAT_STEEL
+	matter = list(MAT_GLASS = MATTER_AMOUNT_REINFORCEMENT)
```
Comment (Techhead0) [inline code/modules/clothing/head/helmet.dm:28]: Was this supposed to be in the helmet/space entry?

File: code/modules/clothing/spacesuits/spacesuits.dm [hunk 4]
```diff
@@ -32,6 +32,8 @@
 
 	var/obj/machinery/camera/camera
 	var/tinted = null	//Set to non-null for toggleable tint helmets
+	origin_tech = "{'materials':1}"
```
Comment (Techhead0) [inline code/modules/clothing/spacesuits/spacesuits.dm:35]: Lower-tech than intended?

File: code/modules/clothing/head/helmet.dm (hunk 3 above)
Comment (biscuitWizard) [reply code/modules/clothing/head/helmet.dm:28]: Nope

File: code/modules/clothing/spacesuits/spacesuits.dm (hunk 4 above)
Comment (biscuitWizard) [reply code/modules/clothing/spacesuits/spacesuits.dm:35]: Tell me what to put, I thought materials 1 made sense.

Comment (Techhead0) [reply code/modules/clothing/spacesuits/spacesuits.dm:35]: It just feels weird that the spaceworthy helmet is lower tech than the physical-protection-only one, that's all

**Review by comma — APPROVED** [review]

**Pull request merged**